pennylane/drawer/tape_text.py

```diff
@@ -89,6 +160,60 @@ def _add_measurement(m, layer_str, wire_map, decimals, cache):
     return layer_str
 
 
+def _find_mid_measure_cond_connections(operations, layers):
```

Comment:
Is this code we will be able to re-use for the mpl drawer too? If so, it might be worth moving this to a different file to be used by both the text and mpl drawer.

Reply:
Good point, I'll move it to `utils.py`